package.json

```diff
@@ -4,9 +4,8 @@
   "description": "A Fieldmanager extension to render dynamic content in modal boxes/overlays on a site.",
   "private": true,
   "scripts": {
-    "watch": "npm run start",
-    "start": "webpack --config client/config/webpack.config.js -d --watch --colors",
-    "build": "webpack --config client/config/webpack.config.js -p"
+    "dev": "webpack --config client/config/webpack.config.js -d --mode development",
```

Comment:
it doesn't look like this does anything special based on mode from the webpack config, do you need it?

Reply:
No, you're right. I actually just copied this in at the start but didn't actually set up the dev task.